storage/store.go

```diff
@@ -1125,10 +1140,11 @@ func (s *Store) ExecuteCmd(args proto.Request, reply proto.Response) error {
 // Resolved flag to true so the client retries the command
 // immediately. If the push fails, we set the error's Resolved flag to
 // false so that the client backs off before reissuing the command.
-func (s *Store) resolveWriteIntentError(wiErr *proto.WriteIntentError, rng *Range, args proto.Request,
+func (s *Store) resolveWriteIntentError(ctx context.Context, wiErr *proto.WriteIntentError, rng *Range, args proto.Request,
 	pushType proto.PushTxnType, wait bool) error {
+	ctx = log.Add(ctx, log.Err, wiErr, log.Key, args.Header().Key)
 	if log.V(1) {
-		log.Infof("resolving write intent on %s %q: %s", args.Method(), args.Header().Key, wiErr)
+		log.Infoc(ctx, "resolving write intent")
```

Comment:
yeah, it's already in there - the context is known by the time the command gets routed to the range, so it will be in the context that ends up here.
probably the key should also be set further upstream already. we'll have to have a guideline for which variables you can expect to already be there.